jsk_2013_04_pr2_610/jsk_2013_04_pr2_610.rosinstall

```diff
@@ -0,0 +1,12 @@
+- git:
+    local-name: jsk-ros-pkg/jsk_demos
+    uri: https://github.com/jsk-ros-pkg/jsk_demos.git
+    version: master
+- git:
```

Comment:
@sktometometo please add comment whey we need this package from source .i.e. https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.kinetic#L18 with link to https://github.com/ros/rosdistro/pull/24310

Reply:
added comments for jsk_planning and pr2_gripper_sensor_msgs.
PR for pr2_gripper_sensor_msgs seems to be already merged.